src/utils/api.js

```diff
@@ -37,10 +42,28 @@ function navigateDefinition(directory): NetResult[] {
     .value();
 };
 
+const algoliaSearch = _.debounce((params, cb) => {
```

Comment:
no need to debounce, if you hit into QPS limits, we'll make sure you get enough to make it work.